scala-steward.conf

```diff
@@ -20,9 +20,17 @@ dependencyOverrides = [
    }
 ]
 
-# Ignore Akka updates following licence changes. Note, we hope to provide better
-# solutions to this going forward; this is a crude sticking-plaster to avoid us
-# sleepwalking into lots of fees.
-updates.ignore = [ { groupId = "com.typesafe.akka" }, {groupId = "com.lightbend.akka"} ]
+updates.ignore = [
 
-pullRequests.customLabels = [ "dependencies" ]
\ No newline at end of file
+  # Ignore Akka updates following licence changes. Note, we hope to provide better
+  # solutions to this going forward; this is a crude sticking-plaster to avoid us
+  # sleepwalking into lots of fees.
+  { groupId = "com.typesafe.akka" }, {groupId = "com.lightbend.akka"},
+
+  # Ignore updates to non-LTS versions of the Scala 3 library.
+  # See https://www.scala-lang.org/blog/2022/08/17/long-term-compatibility-plans.html#owners-of-commercial-projects
+  {groupId = "org.scala-lang", artifactId = "scala3-library", version = {prefix = "3.4."}},
+  {groupId = "org.scala-lang", artifactId = "scala3-library", version = {prefix = "3.5."}},
```

Comment:
I wasn't aware of this version `prefix` syntax but I see it's listed in https://github.com/scala-steward-org/scala-steward/blob/main/docs/repo-specific-configuration.md - awesome!

I wonder if it would make sense to change the `com.typesafe.akka` ignores so that we ignore 2.7, 2.8, 2.9, but not 2.6 (as 2.6.x is still licensed under Apache V2, not BSL). Would get slightly more verbose though - it's a shame that Scala Steward does not support a greater-than-or-equal syntax like `version = { ge = "2.7"}`, as that would be more concise.